app/models.py

```diff
@@ -63,6 +63,15 @@ def serialize(self):
             'last_updated': updated
         }
 
+    def serialize_with_vote_direction(self, apikey=None):
```

Comment:
Since we're adding it to the docs that all Resources now have a vote direction, we should just add this logic into the `serialize` function rather than having a separate method for it. Doing it this way doesn't seem to have much benefit, but it does add in the problem of confusion of which method to use when, and also a response that should have the vote direction might just call `serialize` instead on accident. Example: https://github.com/OperationCode/resources_api/blob/d40d585409116f4e74c92c1c062149182a338bf0/app/api/routes/resource_modification.py#L190

Reply:
Okay. This involves touching more files but I guess I can just have `serialize`